src/components/SearchBox.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+const SearchBox = () => {
```

Comment:
You need to call `API` in this component. We don't need to have static files since we have the backend ready. Since you need some data on that page you need to call API that this searchBox

Reply:
let me work on that